src/test/regress/sql/insert_select_repartition.sql

```diff
@@ -0,0 +1,15 @@
+
+CREATE SCHEMA insert_select_repartition;
+SET search_path TO 'insert_select_repartition';
+
+SET citus.shard_count TO 8;
+CREATE TABLE src_table (a int);
+SELECT create_distributed_table('src_table', 'a');
+INSERT INTO src_table SELECT i FROM generate_series(1, 100) i;
+
+SET citus.shard_count TO 3;
+CREATE TABLE trg_table (a int);
```

Comment:
`trg` as opposed to `dst` or `tgt` is misleading since `trg` could be read as `trigger` as much as `target`